### PR TITLE
Batch upsert for all wallets

### PR DESCRIPTION
**Pull request by paullinator**
* Change throttling logic for refreshWallet to batch across all wallets, not per wallet since each dispatch causes a full revalidation of the redux state and rendering pipeline
* Change upsertWallet to allow multiple wallets to be upsert'ed in one call. Reduces the number of dispatches and re-renders.

### CODE REVIEW

**Comment by paullinator** [comment]
Note: this PR plus https://github.com/Airbitz/edge-react-gui/pull/615

Will reduce the time to sync edgy40 account (2nd login) on an iPhone 6 from 45s to 35s

File: src/modules/UI/Wallets/action.js [hunk 1]
```diff
@@ -84,53 +84,63 @@ export const selectWallet = (walletId: string, currencyCode: string) => (dispatc
   }
 }
 
-function dispatchUpsertWallet (dispatch, wallet, walletId) {
-  dispatch(upsertWallet(wallet))
-  refreshDetails[walletId].delayUpsert = false
-  refreshDetails[walletId].lastUpsert = Date.now()
+function dispatchUpsertWallets (dispatch, wallets: Array<EdgeCurrencyWallet>) {
+  global.pcount('dispatchUpsertWallets')
```
Comment (kylanhurt) [inline src/modules/UI/Wallets/action.js]: @paullinator What is `global.pcount` supposed to do? I am unfamiliar with it.

Comment (paullinator) [reply src/modules/UI/Wallets/action.js]: Ack. this snuck in there. It's a debug line that counts the number of calls to that line of code. It's defined in `app.js`. It's safe to leave in, but should be removed.

Comment (paullinator) [reply src/modules/UI/Wallets/action.js]: I'll remove it with a future PR since I'm still in this code.

**Pull request merged**